.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# Reference used: https://circleci.com/docs/2.0/language-android/#docker-images
+
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/code
+
+    docker:
+      - image: circleci/android:api-23-alpha
```

Comment:
Since there is no reason for this to be a particular version, I'd use the latest version. It looks like a bug if it's 23 and the target SDK is 22.